src/test/java/com/virtuslab/archunit/NamingTestSuite.java

```diff
@@ -26,16 +26,6 @@ public void interface_names_should_start_with_I() {
         .check(importedClasses);
   }
 
-  @Test
-  public void class_names_should_not_end_with_Manager() {
```

Comment:
This rule should stay; it's just the exception made for `com.virtuslab.gitmachete.frontend.externalsystem.project.MacheteTaskManager` that can be removed